pkgs/public/matrix/default.nix

```diff
@@ -1,5 +1,6 @@
 {
   matrix-puppet-skype = import ./matrix-puppet-skype;
+  matrix-puppet-discord = import ./matrix-puppet-discord;
```

Comment:
I think this should be `mx-puppet-discord` instead? It doesn't match the others but is the project's name, and the `matrix-puppet-X` prefix is mostly taken by someone else's projects.

Reply:
Ah, didn't realise that. Done :)